ui/uihandler.go

```diff
@@ -472,13 +453,17 @@ func Serve(port int, debugpath string, uiPath string) {
 		}
 		http.HandleFunc(uiPath, Handler)
 		fmt.Printf("UI starting - visit:\nhttp://localhost:%d%s\n", port, uiPath)
-		logoPath = uiPath + logoURI
-		http.HandleFunc(logoPath, LogoHandler)
-		chartJSPath = uiPath + chartjsURI
-		http.HandleFunc(chartJSPath, ChartJSHandler)
-		// Use relative paths after that (in the html template):
-		logoPath = "./" + logoURI
-		chartJSPath = "./" + chartjsURI
 	}
+
+	// Serve static contents in the ui/static dir.
+	// We use directory relative to this file to find the static contents, so no matter where is the working dir, static dir could be found.
+	_, filename, _, ok := runtime.Caller(0)
+	if ok {
+		fs := http.FileServer(http.Dir(path.Join(path.Dir(filename), "static")))
```

Comment:
this would need to be relative

Reply:
Because the go binary can be anywhere, I feel putting the static files in a static location is easier - for both local dev or docker.

Reply:
I'm still concerned that one can't cp the binary around anymore if we go that way. I may get over that :-)
maybe we can make packages

speaking of which - there should probably be an alternate location, like
/usr/local/lib/fortio-0.3.6/static
for when we have packages

Reply:
We have to have statics delivered via some sort of packaging. Otherwise the page cannot scale. HTMLs need to be moved too. 

I'm thinking whether we should just have static HTML and JS. Then use the JSON API to retrieve data from browser

Reply:
yes some ajaxy thing would be good, we should meet to talk about plans for that

but I'm still a bit attached to self contained binary ... working on getting over it if there is a benefit though